### PR TITLE
Changelog proposition

### PR DESCRIPTION
**Pull request by Skejven**
Changelog proposition that should be updated with every significant change. Current version is created as an example. After releasing `1.0.0` all `RC` versions will be removed.

---

I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Unreleased
+List of changes that are finished but not yet released in any final version.
+
+## Version 1.0.0 (*In progress*)
+- Initial open source release
+
+## Version [1.0.0-RC5](https://github.com/Cognifide/knotx/releases/tag/1.0.0-RC5)
+- Changed Knot.x modules naming convention to io.knotx.*.
+- [fixed Form submission](https://github.com/Cognifide/knotx/issues/237).
```
Comment (marcinczeczko) [inline CHANGELOG.md]: I think, better is to use Pull request ID as it effectively provides the changes, so you can later easily see what was changed in the code. Maybe use following syntax:
```
- [PR 123](http://link_to_PR)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Unreleased
+List of changes that are finished but not yet released in any final version.
+
+## Version 1.0.0 (*In progress*)
+- Initial open source release.
+
+## Version [1.0.0-RC5](https://github.com/Cognifide/knotx/releases/tag/1.0.0-RC5)
+- Changed Knot.x modules naming convention to io.knotx.*.
+- [fixed Form submission](https://github.com/Cognifide/knotx/issues/237).
```
Comment (marcinczeczko) [inline CHANGELOG.md]: Use following syntax:
```
- (PR 123)[https://link_to_PR] - Short description of the changes
```

Comment (Skejven) [inline CHANGELOG.md]: Changed. Updated also `CONTRIBUTING.md`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -33,6 +33,11 @@ This documentation after update is ported to [Knot.x wiki](https://github.com/Co
 When updating documentation please update proper markdown pages in [documentation](https://github.com/Cognifide/knotx/tree/master/documentation) module following [instructions](https://github.com/Cognifide/knotx/blob/master/documentation/README.md) and include it with your pull request.
 After your pull request is merged, wiki pages will be updated. **Please do not update wiki pages directly because your changes will be lost.**
 
+## Changelog
+When changing or fixing some important part of Knot.x, please remember to update [Changelog](https://github.com/Cognifide/knotx/blob/master/CHANGELOG.md).
+Your entry should be enlisted in `Unreleased` section. It will be moved to appropriate release notes when released.
+Please use convention `PR-X - description`, where `PR-X` links to a pull request with changes.
```
Comment (marcinczeczko) [inline CONTRIBUTING.md]: You can show the markdown code of that entry:
```
`+- [PR-ABC](https://github.com/Cognifide/knotx/pull/ABC) - short description of the PR.`
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Unreleased
+List of changes that are finished but not yet released in any final version.
+
+## Version 1.0.0 (*In progress*)
+- Initial open source release.
+
+## Version [1.0.0-RC5](https://github.com/Cognifide/knotx/releases/tag/1.0.0-RC5)
+- Changed Knot.x modules naming convention to io.knotx.*.
+- [PR-237](https://github.com/Cognifide/knotx/issues/238) - fixed Form submission.
```
Comment (marcinczeczko) [inline CHANGELOG.md]: knotx/pull/238

File: CONTRIBUTING.md (hunk 3 above)
Comment (Skejven) [inline CONTRIBUTING.md]: Fixed.

Comment (Skejven) [inline CONTRIBUTING.md]: Fixed.

File: CHANGELOG.md (hunk 4 above)
Comment (Skejven) [inline CHANGELOG.md]: Fixed.

**Pull request merged**